app/services/cfe/submission_manager.rb

```diff
@@ -58,22 +28,12 @@ def submission
       @submission ||= Submission.create!(legal_aid_application_id:)
     end
 
-    def call_common_services
-      COMMON_SERVICES_V5.each do |service|
-        make_logged_call_to service
-      end
-    end
-
-    def call_non_passported_services
-      return if submission.passported?
+  private
 
-      NON_PASSPORTED_SERVICES.each do |service|
-        make_logged_call_to service
-      end
+    def services
+      ServiceSet.call(submission)
```

Comment:
This is really semantics and not a big deal, but I wonder if it might be helpful if the `ServiceSet` responded to `.for` instead of `.call` —since `.call` feels kind of arbitrary to me.

```ruby
def services
  ServiceSet.for(submission: submission)
end
```

I appreciate we might prefer every "service object" to be invoked in the same way though.

Reply:
I see the advantage, but prefer the consistent use of `.call` myself